Examples/Io/Performance/ActsExamples/Io/Performance/VertexPerformanceWriter.cpp

```diff
@@ -673,13 +673,11 @@ ActsExamples::ProcessCode ActsExamples::VertexPerformanceWriter::writeT(
               }
 
               // Save track parameters after the vertex fit
-              const auto paramsAtVtxFitted = propagateToVtx(trk.fittedParams);
-              if (paramsAtVtxFitted != std::nullopt) {
+              if (trk.fittedMomentum != std::nullopt) {
```

Comment:
```suggestion
              if (trk.fittedMomentum) {
```

nitpick: or `has_value` is preferred IMO